pkg/resourcemonitor/resourcemonitor.go

```diff
@@ -32,6 +31,7 @@ import (
 	podresourcesapi "k8s.io/kubelet/pkg/apis/podresources/v1"
 
 	"github.com/jaypipes/ghw"
+	"k8s.io/klog/v2"
```

Comment:
This line should be together with the other k8s dependencies

Reply:
right, fixed.